app/mailers/issue_mailer.rb

```diff
@@ -10,4 +8,11 @@ def issue_created(id)
     mail to: emails,
          subject: 'Подано нову скаргу'
   end
+
+  def issue_status_changed(id)
+    @issue = Issue.find(id)
+    @follower_emails = @issue.followers(User).pluck(:email)
+    return unless @follower_emails.any?
+    mail to: @follower_emails.join(';'), subject: 'Змінено статус скарги'
```

Comment:
this is not OK ... you will show all user emails in one email in "TO" field.
It's better to send to each user same email + you can say in email "Hello #{@user.first_name}"